handel/src/aggregation.rs

```diff
@@ -327,8 +314,11 @@ where
             // next time the timeout triggers the next level needs activating
             self.next_level_timeout += 1;
 
+            let store_rw = self.protocol.store();
+            let store = store_rw.read();
+
             // finally start the level.
-            self.start_level(level);
+            self.start_level(level, &*store);
```

Comment:
```suggestion
            self.start_level(level, &store);
```
The dereference is likely unnecessary.